test/__snapshots__/srcmak.test.ts.snap

```diff
@@ -362,14 +370,29 @@ import jsii
 import jsii.compat
 import publication
 
-__jsii_assembly__ = jsii.JSIIAssembly.load(\\"generated\\", \\"0.0.0\\", __name__[0:-6], \\"generated@0.0.0.jsii.tgz\\")
+__jsii_assembly__ = jsii.JSIIAssembly.load(
+    \\"pythonpackage\\", \\"0.0.0\\", __name__[0:-6], \\"pythonpackage@0.0.0.jsii.tgz\\"
+)
 
 __all__ = [
     \\"__jsii_assembly__\\",
 ]
 
 publication.publish()
 ",
+  "my_python_module/submodule/_jsii/pythonpackage@0.0.0.jsii.tgz": "�      �ko�H�����K���6��D��� ��� �TE�z����5�T����<]Ҫ���ݙ��Ύ��ih��p.;f��y�$�T(��
```

Comment:
This needs to be filtered out

Reply:
Yeah... not really sure why my macbook keeps trying to add them... will try to fix